index.html

```diff
@@ -784,15 +784,15 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 interface to allow controls over the number of
 <a>PerformanceResourceTiming</a> objects stored.</p>
 <p>The recommended minimum number of
-<a>PerformanceResourceTiming</a> objects is 150, though this may be
+<a>PerformanceResourceTiming</a> objects is 250, though this may be
```

Comment:
Slightly tangential nit: I think we need to define a stronger language for the minimum, as I suspect that UAs which will decide to go with lower buffer sizes will be ignored by developers, which will result in dropped entries and potential compat issues. That doesn't have to happen in this PR though.